projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos/aws_demos.uvproj

```diff
@@ -1,2346 +1,3673 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
This project is auto-generated. Please update **vendors/nuvoton/boards/numaker_iot_m487_wifi/CMakeLists.txt** to include the new files.

Reply:
@sarenameas  It's ongoing and in studying how to support two targets in one CMakeLists.

Reply:
@sarenameas  Finished CmakeLists in commit 1b45111. Now, it could auto-gen wifi demo or ethrenet demo.
wifi demo: `cmake -DVENDOR=nuvoton -DBOARD=numaker_iot_m487_wifi -DCOMPILER=arm-keil -S. -Bbuild -G"Unix Makefiles"`
 Eth demo: `cmake -DVENDOR=nuvoton -DBOARD=numaker_iot_m487_wifi -DCOMPILER=arm-keil -S. -Bbuild -G"Unix Makefiles" -DAFR_ENABLE_ETH=1`